setup-readme.sh

```diff
@@ -54,7 +54,7 @@ for d in "$@"; do
 
         if [ "$author" = "ietf" ]; then
             status="Working Group"
-            status_full="IETF [${wgupper} Working Group](https://datatracker.ietf.org/group/${wg}/documents/) Internet-Draft"
+            status_full="IETF [${wgupper} Working Group](https://datatracker.ietf.org/wg/${wg}/documents/) Internet-Draft"
```

Comment:
Robert asked that we use "group" here.  I believe that his plan is to stop using the wg path.

Reply:
```suggestion
            status_full="IETF [${wgupper} Working Group](https://datatracker.ietf.org/group/${wg}/documents/) Internet-Draft"
```